lib/Arch/X86/Runtime/CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ function(add_runtime_helper target_name address_bit_size enable_avx enable_avx51
     BCFLAGS "-std=${required_cpp_standard}"
     INCLUDEDIRECTORIES "${REMILL_INCLUDE_DIR}" "${REMILL_SOURCE_DIR}"
     INSTALLDESTINATION "${REMILL_INSTALL_SEMANTICS_DIR}"
+    TARGET_TRIPLE "x86_64-darwin-macho"
```

Comment:
I think the triple you pass in should be a variable that has been set based on some conditions. Another possibility is that it could be possible to use a pretty generic triple.

Reply:
Yeah that's just temporary to get things building. I think I'm going to make it use `<arch>-linux-elf` or `<arch>-darwin-macho` depending on what the host machine is running.

Reply:
I'm not sure that this cross compilation strategy really works. The runtime has too many references to system headers. For example, even the hyper call enumeration itself relies in `<cstdint>`. 

I was going to try to keep the operating system the same and then just substitute in a different arch. This is because Clang seems to use the host system's headers when cross compiling provided that I'm targeting the same operating system. For example, I'm on an M1 but if I cross compile for `x86_64-darwin-macho`, it uses my system headers. However, this doesn't work for architectures like SPARC because Clang obviously doesn't support `sparc-darwin-macho`. So I'm back to the drawing board.

In order for this to work, I'm going to need to build for `<arch>-none-eabi` and remove any references to system headers in the runtime. It might be best to work on top of that `mps430` branch you mentioned since I see you've already got a solution to the `<cstdint>` includes.

Reply:
OK! Give yourself a limit on how much time you'll spend on it, vs. when it's worth considering if we can get correct-ish results by just using the host architecture target triple (i.e. by not even specifying a triple).